example/index.html

```diff
@@ -33,7 +33,7 @@
         box-shadow: 0 0 30px rgba(0, 0, 0, 0.7);
       }
     </style>
-    <script src="//cdn.bitmovin.com/player/web/8.158.1/bitmovinplayer.js"></script>
+    <script src="//cdn.bitmovin.com/player/web/8.193.0/bitmovinplayer.js"></script>
```

Comment:
Nit: Not sure if this should just be 8 to be honest.

Reply:
Yep, fair point - updated to latest in 255991e8